Elf_Section does not always exists in host <elf.h> (FreeBSD for example)
and its the same as Elf_Half (uint16) on both 32bit and 64bit.

diff -Nur linux-2.6.18.orig/scripts/mod/modpost.h linux-2.6.18/scripts/mod/modpost.h
--- linux-2.6.18.orig/scripts/mod/modpost.h	Fri Sep 22 09:31:42 2006
+++ linux-2.6.18/scripts/mod/modpost.h	Fri Sep 22 09:32:03 2006
@@ -17,7 +17,7 @@
 #define Elf_Shdr    Elf32_Shdr
 #define Elf_Sym     Elf32_Sym
 #define Elf_Addr    Elf32_Addr
-#define Elf_Section Elf32_Section
+#define Elf_Section Elf32_Half
 #define ELF_ST_BIND ELF32_ST_BIND
 #define ELF_ST_TYPE ELF32_ST_TYPE
 
@@ -31,7 +31,7 @@
 #define Elf_Shdr    Elf64_Shdr
 #define Elf_Sym     Elf64_Sym
 #define Elf_Addr    Elf64_Addr
-#define Elf_Section Elf64_Section
+#define Elf_Section Elf64_Half
 #define ELF_ST_BIND ELF64_ST_BIND
 #define ELF_ST_TYPE ELF64_ST_TYPE
 
